tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -5395,10 +5396,16 @@ Thunk::ThunkInfo IrEmitterUnnested::GetThunkInfo(mlir::Operation* op) {
                                     unique_id_attr.getValue().getZExtValue());
   }
   Thunk::ThunkInfo thunk_info(op);
-  thunk_info.profile_annotation = absl::StrFormat(
-      "Thunk:#hlo_op=%s,hlo_module=%s%s#",
-      mlir::mhlo::GetDebugNameFromLocation(op->getLoc()),
-      mlir::mhlo::GetDebugNameFromLocation(module->getLoc()), unique_id_str);
+  if (tsl::profiler::nvtx::RangesEnabled()) {
+    thunk_info.profile_annotation = absl::StrFormat(
+        "hlo_op=%s",
+        mlir::mhlo::GetDebugNameFromLocation(op->getLoc()));
```

Comment:
Here, I didn't add unique_id_str. From my understanding, this just a number that map to the module name.
If it complement the module name and not replace it, I could add it next to the module name instead of the instruction name.

Reply:
yes, somehow even with the long module name, it is not unique, therefore the unique_id_str is used.

Reply:
I found what program_id is. I'll add it to the module name scope. So expect a tiny update to this PR.